RNTester/README.md

```diff
@@ -8,21 +8,15 @@ Before running the app, make sure you ran:
 
     # in this folder
     npm install
-    # and in the toplevel folder
-    cd ..
-    npm install
 
 ### Running on iOS
 
 Mac OS and Xcode are required.
 
-- Start the packager with `npm start` **[important]**
 - Open a new tab in your terminal and start the bucklescript watcher with `npm run watch`
```

Comment:
haha. What I'm generally advocating is to put it in the `npm install` (because I didn't notice it at first :P). The post install script was an additional idea.